File: Data_science_visualisation/pract2.R
# Create Two-dimensional data with the help of data frames and perform different operations on it.

# Load the necessary library for working with data frames
library('dplyr')

# Create a two-dimensional data frame
data <- data.frame(
  A = c(1, 2, 3, 4, 5),
  B = c(6, 7, 8, 9, 10),
  C = c(11, 12, 13, 14, 15)
)

cat("Data Frame:\n")
print(data)

# Accessing columns
cat("Column A:\n")
print(data$A)
cat("Column B:\n")
print(data$B)
cat("Column C:\n")
print(data$C)

# Accessing rows
cat("Row 1:\n")
print(data[1,])


# Sum of columns
cat("Sum of Column A:", sum(data$A), "\n")
cat("Sum of Column B:", sum(data$B), "\n")
cat("Sum of Column C:", sum(data$C), "\n")

# Mean of rows
mean(as.numeric(data[1, ]))
mean(as.numeric(data[2, ]))

# Transpose the data frame
transposed_data <- t(data)
cat("Transposed Data Frame:\n")
print(transposed_data)

File: Data_science_visualisation/pract2.py
# Practical 2:  Create Two-dimensional data with the help of data frames and perform different operations on it.

import pandas as pd

# Two-dimensional data frame
data = pd.DataFrame({'A': [1, 2, 3, 4, 5],
                     'B': [6, 7, 8, 9, 10],
                     'C': [11, 12, 13, 14, 15]})

print("Data Frame:")
print(data)

# Accessing columns
print("Column A:")
print(data['A'])
print("Column B:")
print(data['B'])
print("Column C:")
print(data['C'])

# Accessing rows
print("Row 0:")
print(data.loc[0])
print("Row 1:")
print(data.loc[1])
print("Row 2:")
print(data.loc[2])

# Sum of columns
print("Sum of Column A:", data['A'].sum())
print("Sum of Column B:", data['B'].sum())
print("Sum of Column C:", data['C'].sum())

# Mean of rows
print("Mean of Row 0:", data.loc[0].mean())
print("Mean of Row 1:", data.loc[1].mean())
print("Mean of Row 2:", data.loc[2].mean())

# Transpose the data frame
transposed_data = data.transpose()
print("Transposed Data Frame:")
print(transposed_data)
